streams-contrib/streams-persist-hbase/pom.xml

```diff
@@ -56,6 +56,10 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
theres no preference for jcl or for log4j - whats going on here is jcl is being neutered and routed into slf4j.